package.json

```diff
@@ -56,7 +57,7 @@
     "rxjs": "6.x"
   },
   "devDependencies": {
-    "@openmrs/esm-framework": "next",
+    "@openmrs/esm-framework": "^3.4.1-pre.117",
```

Comment:
Do we want to change from "next" to a specific pre-release?

Reply:
This changed when i run `yarn upgrade` will revert this back to `next`